File: app/src/main/java/com/theone/simpleshare/ui/cocserver/CocServerViewModel.java
package com.theone.simpleshare.ui.cocserver;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class CocServerViewModel extends ViewModel {

    private MutableLiveData<String> mText;

    public CocServerViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("This is dashboard fragment");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: app/src/main/kotlin/com/theone/simpleshare/ui/cocserver/CocServerViewModel.kt
package com.theone.simpleshare.ui.cocserver

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class CocServerViewModel : ViewModel() {
    private val mText: MutableLiveData<String>
    val text: LiveData<String>
        get() = mText

    init {
        mText = MutableLiveData()
        mText.value ="This is dashboard fragment"
    }
}